tests/sentry/api/endpoints/test_organization_details.py

```diff
@@ -323,6 +323,11 @@ def test_is_dynamically_sampled(self):
                 assert not response.data["isDynamicallySampled"]
                 assert "planSampleRate" not in response.data
 
+    def test_dynamic_sampling_custom_target_sample_rate(self):
+        with self.feature({"organizations:dynamic-sampling-custom": True}):
+            response = self.get_success_response(self.organization.slug)
+            assert response.data["targetSampleRate"] == 1.0
+
```

Comment:
nit: it would be good if we add here a test that will check for the response without the feature flag, just in case someone's updates the code and forgets feature flag check, this test would then fail :)

Reply:
Good point!

Reply:
done with b492abfc9a6